### DISCUSSION TITLE
Load data dinamically based on last date from column

### DISCUSSION THREAD

**Discussion by makcim392** [apache/superset#15176] [Q&A / Help]
Hi everyone 😄

I'm showing data from a table, which is shown by dd/mm/yyyy. I want to display by default only data related to the last day the information was updated.

Filter box has an option to show last day, last week, etc. If the data being shown in a dashboard was updated before yesterday, filter box won't cover that date and in consequence the dashboard will come up empty. I want to show by default, only data from the last day the information was updated (which may or may not coincide with Filter box's last day). I fiddled around with JSON properties, etc, but couldn't find anything useful.

**Comment by kamalkeshavani-aiinside** [comment]
I suggest to write a sql query(or custom sql metric) where you can order by your required date column and limit number of rows to show only first result(assuming you want to show fixed number of results from last day).
